### PR TITLE
fix(i18n): t function on withTitle hoc #871

### LINKED ISSUE #871: Fix(i18n): title
**Describe the bug**
Some titles are not translated

**To Reproduce**
1. see https://github.com/dis-moi/extension/pull/814#issuecomment-806011047 

**Expected behavior**
Titles display english text

**Screenshots**
see : https://github.com/dis-moi/extension/pull/814#issuecomment-806011047

### PR DESCRIPTION
**Pull request by gregoirelacoste**
@christpet can you correct my english translation ?
@lutangar i add t function on withTitle HOC. It translate all titles (profiles and extension) but use only extension namespace to refer words (i18n use "extension" NS as defaultNS). It's ok for you ?

closes #871

### CODE REVIEW

File: src/components/atoms/Button/AddNoticeButton/AddNoticeButton.tsx [hunk 1]
```diff
@@ -1,6 +1,7 @@
 import React from 'react';
 import styled from 'styled-components';
 import { Link } from 'components/atoms';
+import { useTranslation } from 'react-i18next';
```
Comment (lutangar) [inline src/components/atoms/Button/AddNoticeButton/AddNoticeButton.tsx:4]: External import should be above our imports.

File: src/app/hocs/withTitle.tsx [hunk 2]
```diff
@@ -1,6 +1,7 @@
 import React, { ComponentType, useLayoutEffect } from 'react';
 import { connect } from 'react-redux';
 import { removeUITitle, setUITitle } from '../content/actions/ui/title';
+import i18next from 'i18next';
```
Comment (lutangar) [inline src/app/hocs/withTitle.tsx:4]: External import should be above our imports.

**Review by lutangar — APPROVED** [review]
`withTitle` is not used by profiles, so I'm not sure I see the issue here.

File: src/i18n/resources/en/extension.json [hunk 3]
```diff
@@ -107,4 +107,16 @@
...
+  "title": {
+    "subscriptions": "$t(menu.subscriptions)",
+    "help": "$t(menu.help)",
+    "filters": "$t(common.filters)",
+    "options": "Options",
+    "about": "$t(menu.about)",
+    "post_form": "Post an information here",
+    "preview": "Preview",
+    "ask_info_form": "Ask an information here",
+    "contribution_details": "Detail of the contribution"
```
Comment (christpet) [inline src/i18n/resources/en/extension.json:120]:
```suggestion
    "post_form": "Post a message here",
    "preview": "Preview",
    "ask_info_form": "Ask for information here",
    "contribution_details": "Message details"
```

**Review by christpet — CHANGES_REQUESTED** [review]
Only a couple of tweaks needed to the language - just made the suggestion.

File: src/components/atoms/Button/AddNoticeButton/AddNoticeButton.tsx (hunk 1 above)
Comment (gregoirelacoste) [reply src/components/atoms/Button/AddNoticeButton/AddNoticeButton.tsx:4]: oops, thx

**Comment by gregoirelacoste** [comment]
> `withTitle` is not used by profiles, so I'm not sure I see the issue here.

ok, good news

**Comment by gregoirelacoste** [comment]
@christpet it's ok for you ?

**Review by christpet — APPROVED** [review]
Looks good!

**Pull request merged**